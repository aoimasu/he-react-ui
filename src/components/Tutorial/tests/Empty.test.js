// @flow
import createTestContext from 'react-cosmos-test/enzyme';
import fixture from '../fixtures/Empty.fixture.js';

const { mount, getWrapper } = createTestContext({ fixture });

beforeEach(mount);

test('<Tutorial /> rendered correctly with empty fixture', () => {
  const wrapper = getWrapper();

  expect(wrapper).toMatchSnapshot();
});
