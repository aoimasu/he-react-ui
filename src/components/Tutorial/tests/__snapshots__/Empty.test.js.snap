// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tutorial /> rendered correctly with empty fixture 1`] = `
<Tutorial
  left={0}
  reversed={false}
  showing="true"
  top={0}
  tutorialStages={null}
/>
`;
